modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-blocking-nodeport-access_{context}"]
+= Blocking external access to NodePort service on a specific host interface
+
+OVN-Kubernetes does not restrict the host interface, therefore NodePort services can be accessed from outside a MicroShift node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
```

Comment:
```suggestion
OVN-Kubernetes does not restrict the host interface, therefore NodePort services can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
```

Reply:
Is it actually NodePort services, or just node ports?

Reply:
The doc in the MicroShift repo said services, lemme confirm as well